dev/test/specs/starter-test.js

```diff
@@ -135,6 +152,8 @@ function createNewArc() {
   browser.switchTab(browser.windowHandles().value[1]);
 }
 
+function expandSuggestionsDrawer() {}
```

Comment:
this function is unused and empty, do you want to remove it?

Reply:
Oops, that shouldn't be there. Thanks!